nucypher/cli/actions.py

```diff
@@ -276,20 +328,22 @@ def make_cli_character(character_config,
 
     # Handle Keyring
 
-    if not dev:
+    if unlock_keyring:
         character_config.attach_keyring()
         unlock_nucypher_keyring(emitter,
                                 character_configuration=character_config,
                                 password=get_nucypher_password(confirm=False))
 
     # Handle Teachers
-    teacher_nodes = load_seednodes(emitter,
-                                   teacher_uris=[teacher_uri] if teacher_uri else None,
-                                   min_stake=min_stake,
-                                   federated_only=character_config.federated_only,
-                                   network_domains=character_config.domains,
-                                   network_middleware=character_config.network_middleware,
-                                   registry=character_config.registry)
+    teacher_nodes = list()
+    if load_preferred_teachers:
+        teacher_nodes = load_seednodes(emitter,
+                                       teacher_uris=[teacher_uri] if teacher_uri else None,
+                                       min_stake=min_stake,
+                                       federated_only=character_config.federated_only,
```

Comment:
Can it be only from config, or can be a CLI option, too?

Reply:
It can be from CLI or from config, with priority given to the CLI flag, which has already been set by this time on the character configuration instance.